modele-social/règles/impôt.yaml

```
@@ -367,19 +367,25 @@ impôt . foyer fiscal . revenu imposable:
       - bénéficiaire . dividendes . imposables
       - autres revenus imposables
 
+impôt . foyer fiscal . revenu imposable . revenu brut:
+  somme:
+    - revenu imposable
+    - (- revenu d'activité abattu)
+    - revenu d'activité
+
+impôt . foyer fiscal . revenu imposable . revenu d'activité:
+  variations:
+    - si: dirigeant = non
+      alors: contrat salarié . rémunération . net imposable
+    - si: entreprise . imposition = 'IS'
+      alors: dirigeant . rémunération . imposable
+
 impôt . foyer fiscal . revenu imposable . revenu d'activité abattu:
   description: |
     Dans le cas général, l'impôt est calculé après l'application d'un abattement forfaitaire fixe. Chacun peut néanmoins opter pour la déclaration de ses *frais réels*, qui viendront remplacer ce forfait par défaut.
-  valeur:
-    nom: assiette
-    variations:
-      - si: dirigeant = non
-        alors: contrat salarié . rémunération . net imposable
-      - si: entreprise . imposition = 'IS'
-        alors: dirigeant . rémunération . imposable
-      - sinon: 0 €/an
+  valeur: revenu d'activité
   abattement:
-    valeur: 10% * assiette
+    valeur: 10%
```

```suggestion
    valeur: 10% * assiette
```

On veut garder `10% * assiette`, car on souhaite plafonner (et mettre un plancher) à l'abattement. Si on écrit uniquement le pourcentage (ce qui est possible avec ce mécanisme), c'est ce dernier qui sera abattu, et non la valeur sur lequel il opère.